chapters/05-version-control-solo.Rmd

```diff
@@ -12,6 +12,115 @@ Clearly identify which tasks are best accomplished by each interface.
   - adding and updating remotes
 - Setting up GitHub for R package (make use of `usethis`), pushing and pulling
 
+## Learning objectives
+
+* identify why version control, specifically Git, is important for software engineering
+* differentiate methods for working with Git and R: RStudio integration, using shell commands in Terminal, `usethis` functions
+* apply the change-stage-commit process as the Git workflow for tracking changes and viewing a repository's commit history
+* create branches to isolate work in progress and protect the main branch while working on major changes
+* publish repositories to GitHub and coordinate between remote and local versions
+
+## Overview
+
+All text (no code):
+* why to use version control
+* why to use Git as a version control system (popularity, GitHub, 
+* how are Git and GitHub different?
+  * local vs remote
+  * publishing 
+* why use version control for software engineering 
+  * basics of version control and why it matters for code
+  * preview this chapter, connect to other chapters
+
+## Git integration in RStudio
+
+### Setting up Git in RStudio
+
+* text: connecting Git and RStudio
+  * accessing Git through Global options
+  * tab in environment pane 
+* code: `usethis::use_git` to initialize repository
+* text: explain what happens when repo is initialized
+  * creates hidden folder containing git history
+  * some files ignored (note that it's possible to add other files to ignore)
+
+### Tracking changes in RStudio
+
+All text and images:
+* exercise: work through basic workflow using RStudio integration
+  * all existing files in repo appear in git pane
+  * ensure box is checked so these files are staged
+  * click Commit button, add commit message and click commit
+  * note pop up box with commit report, files no longer appear in window
+* reiterate workflow by editing README
+  * status difference between tracking new files and editing previously tracked file
+
+* exercise: create new file, commit, delete file, commit
+
+### Viewing history in RStudio
+
+All text and images:
+* click on clock button in Git pane (don't confuse with history tab; all git buttons are in Git tab)
+* explain fields: subject, author, date, SHA
+* click on a previous commit to see changes
+
+## Git in Terminal
+
+* text: git on command line includes commands for full range of tasks, including those we've already covered:
```

Comment:
I'm not sure that we need to do this. There are already lots and lots of resources on learning how to use Git on the terminal, so I don't think we need to duplicate efforts there. Plus, this would be a repetition of the previous section, which I feel would waste some time. I suggest removing this and only doing the basic Git workflow through RStudio. I've personally mostly moved my own Git workflow over to RStudio, so I don't think the learners would lose out on anything here if we don't cover this.

Reply:
I had labeled this as "text" to represent inclusion as a discussion topic (not running the code), to make the segue from the earlier section. If we accept #105 then this becomes moot.